client/src/main/logic/sortingFunctions.ts

```diff
@@ -13,6 +13,15 @@ export function prioritySortDescending(task1: Task, task2: Task): number {
   return task1.priority - task2.priority;
 }
 
+export function searchByNameDescending(
+  searchTerm: string,
+  tasks: Task[]
+): string[] {
+  const matches = tasks.filter(task =>
+    task.title.toLowerCase().includes(searchTerm.toLowerCase())
+  );
+  return matches.map(task => task._id);
+}
```

Comment:
This isn't really a sorting function. Maybe it could go in a different file or be included as a basic function in the file where it is used?

Reply:
True, Ill just move it into the file where its used.